io.github.pieterdd.StretchBreak.yaml

```diff
@@ -0,0 +1,43 @@
+id: io.github.pieterdd.StretchBreak
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: stretch-break
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--device=dri"
+- "--env=G_MESSAGES_DEBUG=none"
+- "--env=RUST_BACKTRACE=1"
+- "--talk-name=org.gnome.Mutter.IdleMonitor"
+- "--talk-name=org.freedesktop.ScreenSaver"
+- "--talk-name=org.kde.ScreenSaver"
```

Comment:
what are these for?

Reply:
Idle detection. The break countdown resets if you're idle long enough. The library that does the idle detection gets its information from `org.gnome.Mutter.IdleMonitor.GetIdletime`, `org.freedesktop.ScreenSaver.GetActiveTime` and a few other fallback places as shown [here](https://github.com/pieterdd/user-idle2-rs/blob/master/src/linux_dbus_impl.rs).

Reply:
is using the portal interface possible? https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Inhibit.html